docs/src/dataDisplay/comment/comment.jsx

```diff
@@ -165,11 +165,41 @@ export default class ElmentsComment extends React.Component {
             isSaveBannerOpen: false,
         };
 
-        this._onMenuClick = this._onMenuClick.bind(this);
-        this._onCloseRemoveBanner = this._onCloseRemoveBanner.bind(this);
-        this._onCloseSaveBanner = this._onCloseSaveBanner.bind(this);
-        this._onRemoveComment = this._onRemoveComment.bind(this);
-        this._onSaveComment = this._onSaveComment.bind(this);
+        this.onMenuClick = this.onMenuClick.bind(this);
+        this.onCloseRemoveBanner = this.onCloseRemoveBanner.bind(this);
+        this.onCloseSaveBanner = this.onCloseSaveBanner.bind(this);
+        this.onRemoveComment = this.onRemoveComment.bind(this);
+        this.onSaveComment = this.onSaveComment.bind(this);
+    }
+
+    onCloseRemoveBanner() {
+        this.setState({ isRemoveBannerOpen: false });
+    }
+
+    onCloseSaveBanner() {
+        this.setState({ isSaveBannerOpen: false });
+    }
+
+    onRemoveComment() {
+        this.setState({ isRemoveBannerOpen: true }, () => {
+            setTimeout(() => this.setState({ isRemoveBannerOpen: false }), 2000);
+        });
+    }
+
+    onSaveComment(id, updatedComment) {
+        const updatedState = { isSaveBannerOpen: true };
+
+        if (id === 1) updatedState.editableCommentText = updatedComment;
+        else if (id === 3) updatedState.editableComment2Text = updatedComment;
+
+        this.setState(updatedState, () => {
+            setTimeout(() => this.setState({ isSaveBannerOpen: false }), 2000);
+        });
+    }
+
+    onMenuClick(id, menuIsOpen) {
+        // eslint-disable-next-line no-console
+        console.log(`Action menu for comment ${id} clicked!  Menu open: ${menuIsOpen}`);
     }
```

Comment:
Hmm, bad merge?  Two `onMenuClick()`s now.

Reply:
NM.  The one above is in the example source code string constant (doing it old-school).  Sorry about the false alarm.